src/sound.cpp

```diff
@@ -599,11 +599,13 @@ void stop_UI_sound()
 
 void play_music_once(const std::string& file)
```

Comment:
`play_music_once` is now the only place where the non-config factory overload is used. Debating just making this use the config overload... would allow me to remove the second ctor too.